pkgs/by-name/ch/chawan/package.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, makeBinaryWrapper
+, curlMinimal
+, man
+, ncurses
+, nim
+, pandoc
+, perl
+, pkg-config
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "chawan";
+  version = "20240301";
```

Comment:
it looks like chawan doesn't have any tags, so version should be  `0-unstable-2024-03-01`

Reply:
Done